### PR TITLE
License Back

### PR DESCRIPTION
**Pull request by schaefjw**
Add the option to print license backs with car photos.

### CODE REVIEW

File: website/print/docs/racer/license-back/.document.inc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+class DriversLicenseBackDocument extends PrintableRacerDocument {
+  function name() { return "Driver's License Back"; }
+
+  function get_available_options() {
+    return array(
+      'use_photo' => array('type' => 'bool',
+                           'desc' => 'Include racer\'s photo',
+                           'default' => true),
+      'use_barcode' => array('type' => 'bool',
```
Comment (jeffpiazza) [inline website/print/docs/racer/license-back/.document.inc]: These two options (use_barcode and date_of_issue) should be removed, please, as they aren't meaningful for this document.

File: website/print/docs/racer/license-back/.document.inc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    $this->layout->SetVerticalGap(0.1);
+  }
+
+  function DrawOneAt($x, $y, &$racer) {   
+    $this->pdf->RoundedRect(11.0-$x-3.5, $y, 3.5, 2.5, 0.125);
+
+    $this->DrawBanner($x, $y);
+
+    if (!$this->get_option('use_photo')) {
+    } else 	if ($racer['carphoto']) {
```
Comment (jeffpiazza) [inline website/print/docs/racer/license-back/.document.inc]: Fix formatting, please.

**Review by jeffpiazza — CHANGES_REQUESTED** [review]
Cool!  Thanks for this!

**Comment by schaefjw** [comment]
This is my first time using GitHub so hopefully I applied all your suggestions correctly.  Thanks for making a great program, the kids loved the licenses!

**Comment by jeffpiazza** [comment]
Looks great!  Thanks!

**Pull request merged**